sdk/data/azcosmos/cosmos_error.go

```diff
@@ -39,16 +31,17 @@ func newCosmosError(response *http.Response) error {
 		return err
 	}
 
-	cError := cosmosError{
-		rawResponse: response,
+	cError := CosmosError{
```

Comment:
This could create an azcore.ResponseError instead.

Reply:
Fixed